.changeset/three-lamps-tell.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I think we should add `@itwin/presentation-hierarchy-builder` to ignore list in changesets config until we are ready to start versioning this package.

Reply:
tried and turns out it's not that trivial - I'll look into that separately, outside of this PR's scope